### PR TITLE
Add details on index cases map tool tip

### PR DESCRIPTION
**Pull request by ciremusyoka**
Part of #1250
 Includes Case number, Case Classification, and Date of Diagnosis details on index cases map tool tip

### CODE REVIEW

**Comment by moshthepitt** [comment]
@ciremusyoka lets get a screenshot if possible please

**Comment by ciremusyoka** [comment]
Screenshot
map-tool-tip

File: src/store/ducks/opensrp/indexCasesDetails/index.ts [hunk 1]
```diff
@@ -0,0 +1,154 @@
+import {
```
Comment (moshthepitt) [inline src/store/ducks/opensrp/indexCasesDetails/index.ts:1]: I think the intention was to have the opensrp directory include only ducks that have entities that come from the OpenSRP API.  I would move this into a "data-warehouse" directory.  We may need to audit the opensrp ducks directory to see what we else we should move - could you create an issue for that please?

Comment (ciremusyoka) [reply src/store/ducks/opensrp/indexCasesDetails/index.ts:1]: Created #1358

**Review by moshthepitt — APPROVED** [review]

**Review by pld — APPROVED** [review]

**Pull request merged**